solutions/monitoring_platforms_intro/index.asciidoc

```diff
@@ -0,0 +1,131 @@
+//Platform=Azure
+//Maturity level=Advanced
+
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+== Monitoring
+=== Context & Problem
+==== Terminology
+
+_Used definition: A monitoring solution helps the monitoring consumer achieve the satisfactory level of control of a defined service_. (https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/manage/monitor/observability[Link])
+
+This definition already includes the following:
+
+* *Defined service:* The resources you want to monitor aka monitored resources. The resources to be monitored can be split in infrastructure and applications on top. 
+* *Level of control:* That is your bandwidth in which your defined service operates normally aka known as baseline
+* *Measuring:* A measurement is a single act that quantifies an attribute of a part, equipment, service or process (CPU load, available memory etc.). Data measured is emitted by the monitored resources and aka telemetry. 
+* *Monitoring consumer:* The user trying to keep the service within its baseline boundaries.
++
+--
+The key to achieve that is a single control plane is usually preferred to simplify the operations for the consumer aka monitoring plane. The relevant content depends on the perspective of the consumer such as performance, costs, compliance and health. Performance in this pattern includes the following (https://github.com/uglide/azure-content/blob/master/articles/best-practices-monitoring.md[Link]):
```

Comment:
Use meaningful names for the links instead of just writing "Link". Or write something like: "... as described in guide xyz/here..." and then use this as the link.